src/tools/cephfs/top/cephfs-top

```diff
@@ -179,6 +180,27 @@ class FSTop(object):
             raise FSTopException(f'error in \'perf stats\' query: {out}')
         return json.loads(buf.decode('utf-8'))
 
+    def items(self, item):
+        if item == "CAP_HIT":
+            return "chit"
```

Comment:
Should we start documenting these? We should have documented it earlier, however, now these metrics are shown in abbreviated form. Maybe in cephfs-top man page?

Reply:
Yeah, make sense and agree.
I will try to doc it.